deluge/core/torrentmanager.py

```diff
@@ -111,10 +111,26 @@ def __init__(self,
         self.move_completed = move_completed
         self.move_completed_path = move_completed_path
 
+    def __eq__(self, other):
+        if not isinstance(other, TorrentState):
```

Comment:
Fair point. I do prefer a positive isinstance statement and you could do it in one line:

```python    
def __eq__(self, other):
    return isinstance(other, TorrentState) and self.__dict__ == other.__dict__
```